www/app/auto/auto.controller.js

```diff
@@ -160,31 +130,28 @@
 
 			function submit() {
 				var autoScore = {
-					fuelPoints: 0,
-					rotorPoints: 0,
-					basePoints: 0,
-					gearTotal: 0,
-					gearAttempt: vm.matchProperties.gearAttempt,
-					rotorTotal: 0,
+                    //fuelpoints
```

Comment:
@Anemotionaltaco why after removing all the other commented out did you leave this in here?

Reply:
I was using it as a reference when I was working on the points but I guess I forgot to delete it